### PR TITLE
Prefix commands with plugin name suffix with action

### PR DESCRIPTION
**Pull request by undg**
I'm finding original commands names not very friendly for autocompleation user like me. Prefixing command with common name and suffixing with action is more autocompleation friendly. 

image

In this PR 4 commands are added. `TwoSlasheshQueries[Disable|Enable|Inspect|Remove]`. To not introduce any breaking changes, original commands (`[Disable|Enable|Inspect|Remove]TwoSlasheshQueries`) are still accessible as a fallback.

### CODE REVIEW

**Comment by marilari88** [comment]
@undg thank you for your contribution. I appreciate it very much 😊. I think your PR definitely makes sense. The commands you propose are more intuitive. I noticed that we have a problem with formatting the code, as there is no stylua configured yet. I would ask you to reset lua/twoslash-queries/init.lua because it looks like you haven't changed anything in it.

**Comment by undg** [comment]
@marilari88 quick response, thats cool!

> I would ask you to reset lua/twoslash-queries/init.lua because it looks like you haven't changed anything in it.

Yes, you are right. It should not be part of the same PR - reverted.

BTW I've updated README.md with new commands. I wasnt touching screencast thought.

File: plugin/twoslash-queries.lua [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
 
-vim.api.nvim_create_user_command("RemoveTwoslashQueries", function()
+vim.api.nvim_create_user_command("TwoslashQueriesRemove", function()
 	require("twoslash-queries").remove_queries()
 end, { nargs = 0, desc = "Remove all two slash queries in the current buffer" })
+
+-- Fallback command for backward compatibility
+vim.api.nvim_create_user_command("EnableTwoslashQueries", function()
+	print("EnableTwoslashQueries commad is obsolete. Use TwoslashQueriesEnable")
+	vim.api.nvim_command("EnableTwoslashQueries")
```
Comment (marilari88) [inline plugin/twoslash-queries.lua]:
```suggestion
	vim.api.nvim_command("TwoslashQueriesEnable")
```

File: plugin/twoslash-queries.lua [hunk 2]
```diff
@@ -1,30 +1,51 @@
...
+end, { nargs = 0, desc = "[Deprecated] Enable two slash queries" })
+
+vim.api.nvim_create_user_command("DisableTwoslashQueries", function()
+	print("DisableTwoslashQueries commad is obsolete. Use TwoslashQueriesDisable")
+	vim.api.nvim_command("DisableTwoslashQueries")
```
Comment (marilari88) [inline plugin/twoslash-queries.lua]:
```suggestion
	vim.api.nvim_command("TwoslashQueriesDisable")
```

File: plugin/twoslash-queries.lua [hunk 3]
```diff
@@ -1,30 +1,51 @@
...
+end, { nargs = 0, desc = "[Deprecated] Disable two slash queries" })
+
+vim.api.nvim_create_user_command("InspectTwoslashQueries", function()
+	print("InspectTwoslashQueries commad is obsolete. Use TwoslashQueriesInspect")
+	vim.api.nvim_command("InspectTwoslashQueries")
```
Comment (marilari88) [inline plugin/twoslash-queries.lua]:
```suggestion
	vim.api.nvim_command("TwoslashQueriesInspect")
```

File: plugin/twoslash-queries.lua [hunk 4]
```diff
@@ -1,30 +1,51 @@
...
+end, { nargs = 0, desc = "[Deprecated] Inspect two slash queries" })
+
+vim.api.nvim_create_user_command("RemoveTwoslashQueries", function()
+	print("RemoveTwoslashQueries commad is obsolete. Use TwoslashQueriesRemove")
+	require("twoslash-queries").remove_queries()
```
Comment (marilari88) [inline plugin/twoslash-queries.lua]:
```suggestion
	vim.api.nvim_command("TwoslashQueriesRemove")
```

**Review by marilari88 — APPROVED** [review]
Good job! We can merge it

**Pull request merged**